rerun_py/rerun_sdk/rerun/__init__.py

```diff
@@ -234,7 +234,8 @@ def init(
         Should we initialize the logging for this application?
     strict
         If `True`, an exceptions is raised on use error (wrong parameter types, etc.).
-        If `False`, errors are logged as warnings instead.
+        If `False`, errors are logged as warnings instead. This is the default.
```

Comment:
```suggestion
        If `False`, errors are logged as warnings instead.
```

well, no it's no longer the default and it's good this way

Reply:
If you don't specify anything or set RERUN_STRICT, strict will be false.

Is there a better way to articulate that?

Reply:
ah. maybe adjust the thing below instead then?
`If unset, this will default to `False` unless the RERUN_STRICT environment variable is set.`